plainly-plugin/src/ui/components/form/UploadForm.tsx

```diff
@@ -1,53 +1,71 @@
 import FormData from 'form-data';
-import { useState } from 'react';
```

Comment:
next bug:

* uploaded
* changed project, save
* move to edit, not working

![image](https://github.com/user-attachments/assets/b3acc616-80e3-43ff-856a-ad1ae40cb542)

Reply:
when I re-enter the form, it shows that project is there, but out of date:

![image](https://github.com/user-attachments/assets/ec3dd805-80a7-4a4d-812d-826076787deb)

Reply:
same, now no maring

![image](https://github.com/user-attachments/assets/02ecccbd-af10-48ab-ae3f-1ea2313a4f2b)